### ISSUE TITLE
There is no way to deliver properly a configured product

### ISSUE THREAD

**Issue by nhomar** [odoo/odoo#28377]
Impacted versions:
12.0
Steps to reproduce:
- Ensure sale_stock is installed.
- With demo data add a configured product.

![image](https://user-images.githubusercontent.com/338634/47961126-2f045100-dfca-11e8-9d81-866b142cc628.png)

- Go to deliveries.

![image](https://user-images.githubusercontent.com/338634/47961136-5d822c00-dfca-11e8-8273-207f2bc95881.png)

Current behavior:

- The **custom** information is not there the only way to have it is going back to the sale order.

Expected behavior:

- Have a way to know the custom information.

Video/Screenshot link (optional):

**Comment by pedrobaeza** [comment]
Yes, we miss that also, and that's why we have created the module: https://github.com/OCA/stock-logistics-reporting/pull/45

**Comment by nhomar** [comment]
Hello @pedrobaeza .

Thanks for point me that out.

I think that report solves partially the problem, but not all of it, let's wait for odoo's answer because  think there is much more to do.

Regards.

**Comment by seb-odoo** [comment]
We know about the problem, but it cannot be fixed in 12.0 because proper fields are missing on the models.

You need to add the value for is_custom values, and also for no_variant attributes.

**Comment by Yenthe666** [comment]
@seb-odoo thanks for the feedback. Is there an internal task for this to do it in V13 or up (or is it planned to be added)? Then this can be closed.

**Comment by seb-odoo** [comment]
@fgi-odoo do we have a task to fix this already?

**Comment by fgi-odoo** [comment]
Yes we plan to improve that as well as in manufacturing orders.

**Comment by C3POdoo** [comment]
Dear @nhomar,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
